apps/prairielearn/src/pages/studentAssessment/studentAssessment.html.ts

```diff
@@ -215,14 +215,15 @@ function GroupCreationJoinForm({
           ? html`
               <div class="col-sm bg-light py-4 px-4 border">
                 <form id="create-form" name="create-form" method="POST">
-                  <h6>Group name</h6>
+                  <label for="groupNameInput">Group name</label>
                   <input
                     type="text"
                     class="form-control"
                     id="groupNameInput"
                     name="groupName"
                     maxlength="30"
                     placeholder="e.g. teamOne"
+                    aria-label="Group name"
```

Comment:
You currently have both a `<label>` and an `aria-label` for the same input. This is redundant.